coalib/bearlib/languages/definitions/Haskell.py

```diff
@@ -0,0 +1,16 @@
+from coalib.bearlib.languages.Language import Language
+
+
+@Language
+class Haskell:
+    extensions = ('.hs')
```

Comment:
Again, you made the same mistake as https://github.com/coala/coala/pull/5748#discussion_r216180172

Reply:
First you say make the tuple of this. When I make tuple then you say previous one is right .